cmd/machine.go

```diff
@@ -87,6 +87,10 @@ func main() {
 		commands.ClosePluginServers()
 		return nil
 	}
+	app.After = func(c *cli.Context) error {
```

Comment:
Got picked up when I rebased against your branch - not sure what's the status of this is?